apps/web/src/App.tsx

```diff
@@ -246,4 +270,34 @@ function AuthHandlerComponent({ children }: { children: React.ReactNode }) {
   );
 }
 
+function SpotLightProvider({ children }) {
```

Comment:
Mantine did not have support for sort order can be good if we like to feature some specific command sometime 😄  or just like them to have them in a specific order...